next/components/forms/ProgressBar.tsx

```diff
@@ -0,0 +1,35 @@
+import cx from 'classnames'
+import {useProgressBar} from 'react-aria'
```

Comment:
I see this isn't formated according to eslint rules - please fix your eslint setup and make sure you have 'format on save' enabled

Reply:
'yarn lint' doesn't give me any errors or warnings for this file, my IDE already have enabled eslint rules